packages/manager/apps/cloud/client/app/dbaas/logs/translations/Messages_fr_FR.json

```
@@ -642,7 +639,10 @@
   "logs_index_modal_suffix": "Suffixe",
   "logs_index_suffix_description": "Seuls les minuscules de a à z, les nombres et les caractères - ou _ sont autorisés.",
   "logs_index_modal_description": "Description",
-  "logs_index_modal_select": "Choisir l'option liée",
+  "logs_index_modal_nb_shard": "Nombre de partition",
+  "logs_index_modal_nb_shard_help": "Une partition dispose d'une capacité de 25 Gio, son prix unitaire est de ",
+  "logs_index_modal_nb_shard_help2": "HT / mois. Le prix par Gio stocké est de ",
+  "logs_index_modal_nb_shard_help3": "HT / mois.",
```

Having translations chunk is not the easiest to translate. 
```suggestion
  "logs_index_modal_nb_shard_help": "Une partition dispose d'une capacité de 25 Gio, son prix unitaire est de {{ unitaryPrice }} HT / mois. Le prix par Gio stocké est de {{ price }} HT / mois.",
```
Using interpolated parameters would be better (https://angular-translate.github.io/docs/#/api/pascalprecht.translate.filter:translate)

---

I tried without success: `€` given by billing API is HTML escaped when interpolated.

---

Fixed with the help of @FredericEspiau